docs/breez-sdk/src/guide/testing.md

```diff
@@ -0,0 +1,31 @@
+# Testing and Development
+
+## Regtest Network
+
+For most testing and development, we recommend using the **regtest network** - a deployed test network maintained by Lightspark that provides a safe environment with no real value at risk.
```

Comment:
Suggestion because the SDK is not dangerous to use, even on mainnet.

```suggestion
For most testing and development, we recommend using the **regtest network** - a deployed test network maintained by Lightspark that is free to use and carries no real-world value.
```

Reply:
Makes total sense, thanks